src/Sinch/Core/Http.cs

```diff
@@ -263,13 +263,19 @@ bool HasNonNullValue(PropertyInfo x)
                 // NOTE: there wil probably be other files supported in the future
                 if (result.IsPdf())
                 {
-                    if (typeof(TResponse) != typeof(Stream))
+                    if (typeof(TResponse) != typeof(ContentResult))
```

Comment:
I don't understand how the type of response would be different from the previous state where it was a Stream. Unless I'm mistaken, the `ContentResult` object will be created on lines 274-278 below, right?

Reply:
Yes, ContentResult will be created below. The response as read as a raw stream, but now it's contained in `ContentResult` class. It's a safe measure when SDK dev will try to use another class which is not compatible with raw stream content.